monkey/monkey_island/cc/services/config_schema/ransomware.py

```diff
@@ -2,32 +2,45 @@
     "title": "Ransomware",
     "type": "object",
     "properties": {
-        "directories": {
-            "title": "Directories to encrypt",
+        "encryption": {
+            "title": "Encryption",
             "type": "object",
             "properties": {
-                "linux_dir": {
-                    "title": "Linux encryptable directory",
-                    "type": "string",
-                    "default": "",
-                    "description": "Files in the specified directory will be encrypted "
-                    "using bitflip to simulate ransomware.",
+                "should_encrypt": {
+                    "title": "Encrypt files",
+                    "type": "boolean",
+                    "default": True,
+                    "description": "Selected files will be encrypted using bitflip to simulate "
+                    "ransomware. Enter target directories below.",
                 },
-                "windows_dir": {
-                    "title": "Windows encryptable directory",
-                    "type": "string",
-                    "default": "",
-                    "description": "Files in the specified directory will be encrypted "
-                    "using bitflip to simulate ransomware.",
+                "directories": {
+                    "title": "Directories to encrypt",
+                    "type": "object",
+                    "properties": {
+                        "linux_dir": {
+                            "title": "Linux encryptable directory",
+                            "type": "string",
+                            "default": "",
+                            "description": "Files in the specified directory will be encrypted "
+                            "using bitflip to simulate ransomware.",
```

Comment:
"Ransomware encryption will be simulated by flipping every bit in the files contained within the specified directory"